src/test/java/com/snowflake/kafka/connector/internal/FileNameUtilsTest.java

```diff
@@ -154,4 +160,41 @@ public void testFileNameWontSupportMoreThan32767Partitions() {
                     endOffset))
         .isInstanceOf(IllegalArgumentException.class);
   }
+
+  @Test
+  public void testPrepareFilesOffsetsLogString() {
```

Comment:
could we split the test into 3 separate tests as we have here at least 3 different test cases?